nui_shared_utils/__init__.py

```
@@ -1,252 +1,263 @@
-    # Lambda context helpers
-    "get_lambda_environment_info",
-    # JWT authentication
-    "validate_jwt",
-    "require_auth",
-    "check_auth",
-    "get_jwt_public_key",
-    "JWTValidationError",
-    "AuthenticationError",
-]
+
+__all__ = sorted(set(_LAZY_EXPORTS) | {"slack_setup"})
```

Ruff (0.15.12)

[error] Invalid format for `__all__`, must be `tuple` or `list`

(PLE0605)